web/src/styles/base/height-width.scss

```diff
@@ -22,6 +22,10 @@
   min-height: 38rem;
 }
 
+.minh-2606 {
```

Comment:
TIL about [Sass Functions](https://sass-lang.com/documentation/at-rules/function/). It looks like we only have 5 classes like this right now, but if we see ourselves adding or needing more in the future, maybe this is something we could be programmatically generate so they're either already available or could be extended further.

Reply:
+1 By programmatically generating these classes, we can ensure future flexibility and consistency in our codebase as we scale.

Reply:
Spoke to JP, we are going to hold off on implementing some of the features of SASS given the limited number of css that would change. Also, after doing some research, it looks like the best approach would have been to create a list and then iterate over the list using @ each, and defining the css